docker/CMSRucioClient/traces/collect_send_traces.py

```diff
@@ -0,0 +1,161 @@
+#! /usr/bin/env python3
+
+from __future__ import absolute_import, division, print_function
+
+import io
+import json
+import os
+import time
+
+import requests
+#
+def send_trace(trace, trace_endpoint, user_agent, retries=5):
+    """
+    Send the given trace to the trace endpoint
+    :param trace: the trace dictionary to send
+    :param trace_endpoint: the endpoint where the trace should be send
+    :param user_agent: the user agent sending the trace
+    :param retries: the number of retries if sending fails
+    :return: 0 on success, 1 on failure
+    """
+    if user_agent.startswith('pilot'):
+        return 0
```

Comment:
I copied this method from Rucio repository so I don't have to depend on the rucio library because this is very simple one. I guess if this is a pilot run, no traces need to be collected?  Is this a Atlas thing?  If we need to run something that don't need to collect traces, we can use it too.